tuning/utils/config_utils.py

```diff
@@ -135,3 +136,34 @@ def txt_to_obj(txt):
     except UnicodeDecodeError:
         # Otherwise the bytes are a pickled python dictionary
         return pickle.loads(message_bytes)
+
+
+def transform_placeholders(template: str) -> str:
```

Comment:
could we please rename this function to be more descriptive? 

sanitise jinja placeholders?